scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -112,10 +123,45 @@ def gen_bkg_list(bkg_path='.', file_type='MOM', yaml_name='bkg.yaml'):
                                   'soca',
                                   'berror',
                                   'parametric_stddev_b.yaml')
-config = {'atm': 'false'}
+config = {}
 ufsda.yamltools.genYAML(config, output=berr_yaml, template=berr_yaml_template)
 
-# generate YAML file for soca_var
+# link yaml for decorrelation length scales
+corscales_yaml = os.path.join(gdas_home,
+                              'parm',
+                              'soca',
+                              'berror',
+                              'soca_setcorscales.yaml')
+ufsda.disk_utils.symlink(corscales_yaml,
+                         os.path.join(stage_cfg['stage_dir'], 'soca_setcorscales.yaml'))
+
+# generate yaml for bump C
+# TODO (Guillaume): move the possible vars somewhere else
+vars3d = ['tocn', 'socn', 'uocn', 'vocn', 'chl', 'biop']
+vars2d = ['ssh', 'cicen', 'hicen', 'hsnon',
+          'swh',
```

Comment:
is there a particular reason why this one var is alone on this line?

Reply:
I was going to say I organized them by domains: ocean, sea-ice. wave and atm-fluxes ... But clearly I didn't.
How about this:
```python
vars3d = ['tocn', 'socn', 'uocn', 'vocn', 
          'chl', 'biop']
vars2d = ['ssh',
          'cicen', 'hicen', 'hsnon',
          'swh',
          'sw', 'lw', 'lw_rad', 'lhf', 'shf', 'us']
```
or I could just move `swh` to L141 ...

Reply:
I would just move swh, seems a bit excessive to sort them for this purpose, but I defer to you